zebra-utils/src/bin/zebra-tip-height/args.rs

```diff
@@ -0,0 +1,24 @@
+//! zebra-tip-height arguments
+//!
+//! For usage please refer to the program help: `zebra-tip-height --help`
+
+#![deny(missing_docs)]
+#![allow(clippy::try_err)]
+
+use std::path::PathBuf;
+
+use structopt::StructOpt;
+
+use zebra_chain::parameters::Network;
+
+/// zebra-tip-height arguments
+#[derive(Debug, StructOpt)]
+pub struct Args {
+    /// Path to Zebra's cached state.
+    #[structopt(short, long, parse(from_os_str))]
```

Comment:
Usability: can we make the state path optional, and use the default path when it's not specified?

Reply:
Done, good idea!